Minesweeper.py

```diff
@@ -351,8 +350,7 @@ def uncover(self, pos):
                     if tile['covered']:
                         tile['covered'] = False
                         if tile['mine']:
-                            self.game.gamestate.game_over = True
-                            self.game.gamestate.game = False
+                            game.gamestate.set_state_game_over()
```

Comment:
still missing a Tile class

Reply:
didn't manage to get it fully implemented before work was done but I got some stuff in there and functioning properly